.github/workflows/publish.yml

```diff
@@ -0,0 +1,80 @@
+name: Publish
+
+on:
+  release:
+    types: [created, published]
```

Comment:
Do you know what's the difference between these two? It doesn't seem to be documented in [the trigger events doc](https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows#release).

Reply:
I'm not 100% sure, but GitHub's UI copy gives us some clue. On the releases page, the button says "Draft a new release". On that page, there are two buttons, "Publish release" or "Save as draft". The former likely corresponds to a `published` event, and the latter may trigger a `created` step.

Here, we're creating a release through the `github/create-release` action, whose name implies a `created` event, but I vaguely recall triggering `published` instead. It's really too bad that there's no easy way to do a dry run of this.